tests/test_python.py

```diff
@@ -1,15 +1,71 @@
-"""Basic tests for the Python 3.6 and 3.9 base container images."""
+"""Basic tests for the Python base container images."""
+
 from bci_tester.data import PYTHON310_CONTAINER
 from bci_tester.data import PYTHON36_CONTAINER
 from bci_tester.data import PYTHON39_CONTAINER
 
+import pytest
+from _pytest.mark.structures import ParameterSet
+from pytest_container import DerivedContainer
+from pytest_container.container import container_from_pytest_param
+
+bcdir = "/tmp/"
+orig = "tests/"
+appdir = "trainers/"
+outdir = "output/"
+appl1 = "tensorflow_examples.py"
+
+# copy tensorflow module trainer from the local applico to the container
+DOCKERF_PY_T = f"""
+WORKDIR {bcdir}
+RUN mkdir {appdir}
+RUN mkdir {outdir}
+COPY {orig + appdir}/{appl1}  {appdir}
+"""
+
+PYTHON36_CONTAINER_T = pytest.param(
+    DerivedContainer(
+        base=container_from_pytest_param(PYTHON36_CONTAINER),
+        containerfile=DOCKERF_PY_T,
+    ),
+    marks=PYTHON36_CONTAINER.marks,
+)
 
+PYTHON39_CONTAINER_T = pytest.param(
+    DerivedContainer(
+        base=container_from_pytest_param(PYTHON39_CONTAINER),
+        containerfile=DOCKERF_PY_T,
+    ),
+    marks=PYTHON39_CONTAINER.marks,
+)
+
+PYTHON310_CONTAINER_T = pytest.param(
+    DerivedContainer(
+        base=container_from_pytest_param(PYTHON310_CONTAINER),
+        containerfile=DOCKERF_PY_T,
+    ),
+    marks=PYTHON310_CONTAINER.marks,
+)
```

Comment:
What about cycling through different python images?

Reply:
I did it in the commit https://github.com/SUSE/BCI-tests/pull/108/commits/08e2851a68034784f6f107fb424232ee83027bc3

Reply:
Done